qemu/tests/cfg/virtio_fs_supp_group_transfer.cfg

```diff
@@ -0,0 +1,51 @@
+- virtio_fs_supp_group_transfer:
+    type = virtio_fs_supp_group_transfer
+    only Linux
+    no Host_RHEL.m8, Host_RHEL.m7, Host_RHEL.m6
+    virt_test_type = qemu
+    required_qemu = [4.2.0,)
+    s390, s390x:
+        required_qemu = [5.2.0,)
+        vm_mem_share = yes
+        pre_command_noncritical = yes
+        pre_command = "echo 3 > /proc/sys/vm/drop_caches"
+        setup_hugepages = yes
+        kvm_module_parameters = 'hpage=1'
+        expected_hugepage_size = 1024
+    clone_master = yes
+    master_images_clone = image1
+    remove_image_image1 = yes
+    kill_vm = yes
+    start_vm = yes
+    filesystems = fs
+    fs_driver = virtio-fs
+    force_create_fs_source = yes
+    remove_fs_source = yes
+    fs_target = 'myfs'
+    fs_driver_props = {"queue-size": 1024}
+    mem = 4096
```

Comment:
For virtiofs, there is no need to specify memory size later. And change  size_mem1 = ${mem}M together.
Thanks.